lib/assessment.js

```diff
@@ -39,7 +71,15 @@ var newArray2 = function newArray2(length, defaultsFunction) {
 // The function you write should return
 // the object passed in as an argument or the object created
 var addMethod = function addMethod(propertyName, method, obj) {
+  if (!obj) {
+    var target = {};
+  } else {
+    var target = obj;
+  }
+
+  target.prototype.propertyName = method;
```

Comment:
Since we're not using a constructor function to make lots of identical objects, we don't have to worry about making lots of identical copies of our methods; consequently, there's not much to be gained by putting the methods in a separate object (i.e. the constructor function's prototype). 

Additionally, since `target` is not a constructor function, it actually doesn't have a `prototype` property, so that line of code may not run.